ops_omgx/docker/vault/vault.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
```

Comment:
are you sure this can be used in a production setup?

Reply:
not going to use inthe PROD deploy, just wanted to make sure the build is green!